marginalia.el

```diff
@@ -70,6 +70,36 @@ It can also be set to an integer value of 1 or larger to force an offset."
   "Use whitespace margin for window widths larger than this value."
   :type 'integer)
 
+(defcustom marginalia-file-age-function #'marginalia--file-age
```

Comment:
There is maybe a bit too much indirection with all these formatter functions. I have to think a bit about this.

Reply:
I get that feeling too, but I also think it's nice to leave the door open to something like the blended colouring I demo'd a while ago, and having the "main" formatting function be a custom variable seems like a somewhat clean way to do this, and then you've got the potential for personal taste with e.g. absolute file formatting and that's what lead to these three formatting functions.

Reply:
Yes, I would like to keep this one variable, but not the others.